### ISSUE TITLE
BUG: No package metadata was found for PyFunceble-dev

### ISSUE THREAD

**Issue by ZeroDot1** [funilrys/PyFunceble#285]
```py
[user@work ~]$ pyfunceble -w 32 -a --logging-level debug -f File.txt 
Traceback (most recent call last):
  File "/usr/bin/pyfunceble", line 33, in <module>
    sys.exit(load_entry_point('PyFunceble-dev==4.0.2b7', 'console_scripts', 'pyfunceble')())
  File "/usr/bin/pyfunceble", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 919, in distribution
    return Distribution.from_name(distribution_name)
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 518, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: No package metadata was found for PyFunceble-dev
```
Reinstall via yay -S 

```py
[user@work ~]$ pyfunceble -w 32 -a --logging-level debug -f File.txt
Traceback (most recent call last):
  File "/usr/bin/pyfunceble", line 33, in <module>
    sys.exit(load_entry_point('PyFunceble-dev==4.0.2b7', 'console_scripts', 'pyfunceble')())
  File "/usr/bin/pyfunceble", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 162, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.10/site-packages/PyFunceble/__init__.py", line 55, in <module>
    import PyFunceble.storage
  File "/usr/lib/python3.10/site-packages/PyFunceble/storage.py", line 58, in <module>
    from box import Box
ModuleNotFoundError: No module named 'box'
[user@work ~]$ 
```
Logfiles: https://drive.google.com/file/d/1gRgWN_wi1N_gf0FzFJsENw4aSgL4yLCV/view?usp=sharing

**Comment by ZeroDot1** [comment]
It feels like a curse, whenever I want to use the program it doesn't work as it should.

Installing the non-DEV version doesn't help this time either.

**Comment by ZeroDot1** [comment]
```py
Traceback (most recent call last):
  File "/usr/bin/pyfunceble", line 33, in <module>
    sys.exit(load_entry_point('PyFunceble==4.0.2', 'console_scripts', 'pyfunceble')())
  File "/usr/bin/pyfunceble", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 162, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.10/site-packages/PyFunceble/__init__.py", line 55, in <module>
    import PyFunceble.storage
  File "/usr/lib/python3.10/site-packages/PyFunceble/storage.py", line 58, in <module>
    from box import Box
ModuleNotFoundError: No module named 'box'

```

**Comment by spirillen** [comment]
> ModuleNotFoundError: No module named 'box'

Your poor thing :smile: 

This seems like the issue and it have happen before and the reason this line was added

https://github.com/funilrys/PyFunceble/blob/cb190310af48830983e8811a92cf974a86b8adb1/requirements.txt#L9


Did you try install with pip, not that i think it might change anything

Else try this one

```
python3.10 -m pip install -I -U --pre pyfunceble-dev
```

The capital `-I` will force all dependencies to be reinstalled.

**Comment by ZeroDot1** [comment]
OK, updating the Python packages manually helped to solve the problem.

**Comment by ZeroDot1** [comment]
![Bildschirmfoto vom 2021-12-15 07-49-28](https://user-images.githubusercontent.com/28985171/146137450-6fc391b1-dabe-450f-b0b4-8b6030220989.png)

**Comment by spirillen** [comment]
Thanks for the reply @ZeroDot1 

Made this PR to ensure a higher version of the box, even this should be done by the installer as the requirements says higher than or equal version=x

**Comment by funilrys** [comment]
This issue is weird ...  @ZeroDot1 is normally installing from Arch Linux Repo which I maintain. I can't explain the solution ...

**Comment by funilrys** [comment]
I assume this should be solved through my Arch Linux update of 5.4.1 (I'm also the maintainer of that Arch Package) ...
